cmd/rollouts-controller/main.go

```diff
@@ -94,6 +95,7 @@ func newCommand() *cobra.Command {
 			defaults.SetAmbassadorAPIVersion(ambassadorVersion)
 			defaults.SetSMIAPIVersion(trafficSplitVersion)
 			defaults.SetAppMeshCRDVersion(appmeshCRDVersion)
+			defaults.SetPrometheusAddress(prometheusAddress)
```

Comment:
Is it possible to do some validation inside `SetPrometheusAddress()` to make sure that the passed value is a URI in the expected format (Scheme, Port Number, ...)?